src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraAcl.scala

```diff
@@ -0,0 +1,26 @@
+
+package org.broadinstitute.dsde.agora.server.business
+
+import org.broadinstitute.dsde.agora.server.business.AgoraPermissions._
+
+object AgoraPermissions {
+  val Read = 1 << 0
+  val Write = 1 << 1
+  val Create = 1 << 2
+  val Redact = 1 << 3
+  val Manage = 1 << 4
+  val ReadWrite = Read | Write
+  val All = Read | Write | Create | Redact | Manage
+}
+
+case class AgoraAcl(permissions: Int) {
+  def canRead: Boolean = (permissions & Read) != 0
+
+  def canWrite: Boolean = (permissions & Write) != 0
+
+  def canCreate: Boolean = (permissions & Create) != 0
+
+  def canRedact: Boolean = (permissions & Redact) != 0
+
+  def canManage: Boolean = (permissions & Manage) != 0
+}
```

Comment:
I was wondering more about stuff like:
1) create AgoraAcl with Read, Write, Create, and Redact
2) given an AgoraAcl, make a new one minus Create and Redact

If we used sets, 1) would be something like acl = Set(Read, Write, Create, Redact), but with the bit field would be acl = AgoraAcl(Read | Write | Create | Redact).

With sets, 2) would be acl = acl - Set(Create, Redact), but with a bit field would be acl = AgoraAcl(acl.permissions & (31 ^ Create) & (31 ^ Redact)) ... I think.

Isn't it easier to work with sets instead of a bit field, especially since we don't need to store the AgoraAcl?